### PR TITLE
Edit MeetingCard view

### PR DESCRIPTION
**Pull request by jimseah-0116**
Swapped the name and meeting in MeetingCard view.
Fixed EditCommand bug.

### CODE REVIEW

**Review by irwynliong — APPROVED** [review]
Lovely brother keep up the good work! 😃

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -208,23 +208,34 @@ Further descriptions on the methods:
...
+### View Meetings feature
+
+#### Implementation
+
+The 'meetings' command allows users to view all the meetings that are scheduled in the current week. The feature is implemented through the `MeetingsCommand` class.
+
+The UI component for this command is the `MeetingsWindow`, which is a pop-up window displaying the meetings for the current week.
+
+#### Design considerations:
+- User can view all meetings scheduled for the current week, *in chronological order*. This allows the user to **efficiently identify** the meetings that are coming up soon.
```
Comment (Greg-Tan) [inline docs/DeveloperGuide.md:235]: Nice explaining your thought process behind your implementation!

File: src/main/resources/view/MeetingListCard.fxml [hunk 2]
```diff
@@ -25,10 +24,9 @@
             <Region fx:constant="USE_PREF_SIZE"/>
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label"/>
+        <Label fx:id="meetingText" text="\$first" styleClass="cell_big_label"/>
```
Comment (Greg-Tan) [inline src/main/resources/view/MeetingListCard.fxml:27]: nice work!

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 3]
```diff
@@ -137,6 +138,10 @@ public void parse_repeatedNonTagValue_failure() {
         // invalid address
         assertParseFailure(parser, validExpectedPersonString + INVALID_ADDRESS_DESC,
                 Messages.getErrorMessageForDuplicatePrefixes(PREFIX_ADDRESS));
+
+        // invalid meeting
+        assertParseFailure(parser, validExpectedPersonString + INVALID_MEETING_DESC,
```
Comment (Greg-Tan) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:143]: good job handling parsing errors

**Review by Greg-Tan — APPROVED** [review]

**Pull request merged**